### DISCUSSION TITLE
Unicorn + "This page is taking too long to load." when viewing issue

### DISCUSSION THREAD

**Discussion by Jongmassey** [community/community#32770] [Projects and Issues]
This issue in a private repo https://github.com/ebmdatalab/sensitive-issues/issues/8 isn't viewable by me or any of my team - could someone advise?

**Comment by 3kh0** [comment]
This happens every so often, here are a few quick fixes that sometimes help.

- Reload the page (easiest in the book)
- Reload and clear cache, in chrome do ctrl+shift+r
- Disconnect from wifi, and reconnect
- Open cmd and do `ipconfig/release` and then `ipconfig/renew`
- Turn off your computer, wait 10 seconds, and turn it back on.
- Try a different browser
- Clear cookies and site data from github.com
- Reboot your router

These are some quick and easy fixes, this issue happens from time to time again, and these fixes seem to fix it for me.

*If this helped please mark this as the answer!* ❤️

**Comment by Jongmassey** [comment]
This was occurring across several machines, running different operating systems and browsers, and several networks. Fundamentally, it's a server-side error (500 series), nothing to do with my local DNS, DHCP, cache.
